modules/ocf/files/puppet-trigger

```diff
@@ -40,13 +46,60 @@ def is_jessie():
         return f.read().strip().startswith('8.')
 
 
-if __name__ == '__main__':
-    parser = argparse.ArgumentParser(description='trigger puppet agent runs')
-    parser.add_argument('-f', '--tail', action='store_true',
-                        help='tail syslog after trigger')
+def switch_to_environment(env):
+    log('Switching environment to "{}".'.format(env))
+
+    hostname = socket.gethostname()
+    ldif = '''\
+dn: cn={hostname},ou=Hosts,dc=OCF,dc=Berkeley,dc=EDU
+changetype: modify
+replace: environment
+environment: {environment}
+'''.format(
```

Comment:
Maybe use `dedent` for this to make it look cleaner (more consistent indentation)?